apps/studio/src/handlers/exportHandlers.ts

```diff
@@ -0,0 +1,120 @@
+import { TableFilter, TableOrView } from "@/lib/db/models";
+import { CsvExporter, Export, JsonExporter, JsonLineExporter, SqlExporter } from "@/lib/export";
+import { ExportStatus } from "@/lib/export/models";
+import { uuidv4 } from "@/lib/uuid";
+import { checkConnection, errorMessages, state } from "./handlerState";
+import rawLog from 'electron-log';
+
+const log = rawLog.scope('ExportHandler');
+
+interface StartExportOptions {
+  table: TableOrView,
+  query?: string,
+  queryName?: string,
+  filters: TableFilter[],
+  exporter: 'csv' | 'json' | 'sql' | 'jsonl'
+  filePath: string
+  options: {
+    chunkSize: number
+    deleteOnAbort: boolean
+    includeFilter: boolean
+  }
+  outputOptions: any
+}
+
+const ExportClassPicker = {
+  'csv': CsvExporter,
+  'json': JsonExporter,
+  'sql': SqlExporter,
+  'jsonl': JsonLineExporter
+}
+
+function getExporter(id: string): Export {
+  const exporter = state.exports.get(id);
+
+  if (!exporter) {
+    throw new Error(errorMessages.noExport);
+  }
+
+  return exporter;
+}
+
+export interface IExportHandlers {
+  'export/add': ({ options }: { options: StartExportOptions}) => Promise<string>,
+  'export/remove': ({ id }: { id: string }) => Promise<void>,
+  'export/removeInactive': () => Promise<string[]>,
+  'export/status': ({ id }: { id: string }) => Promise<ExportStatus>,
+  'export/error': ({ id }: { id: string }) => Promise<string>,
+  'export/name': ({ id }: { id: string }) => Promise<string>,
+  'export/start': ({ id }: { id: string }) => Promise<void>,
+  'export/cancel': ({ id }: { id: string }) => Promise<void>
+}
+
+
+export let exportHandlers = {} as unknown as IExportHandlers;
+
+exportHandlers['export/add'] = async function({ options }: { options: StartExportOptions }) {
```

Comment:
I understand why you did this, but it would be better for it to be a Const (also Uppercase the first word):

```ts
export const exportHandlers extends IExportHandlers = {
  'export/add': ...
}
```
Works just fine, then this can be a const!

![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/279769/ec10b5b3-8b41-471a-8d32-f97fab1658bc)